collector/terminal_services.go

```diff
@@ -0,0 +1,403 @@
+			"Total elapsed time that all of the threads of this process used the processor to execute instructions.",
+			[]string{"session_name"},
+			nil,
+		),
+		PercentUserTime: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "user_time_total"),
+			"Total elapsed time that this process's threads have spent executing code in user mode. Applications, environment subsystems, and integral subsystems execute in user mode.",
+			[]string{"session_name"},
+			nil,
+		),
+		PoolNonpagedBytes: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "pool_non_paged_Bytes"),
+			"Number of bytes in the non-paged pool, an area of system memory (physical memory used by the operating system) for objects that cannot be written to disk, but must remain in physical memory as long as they are allocated. This property displays the last observed value only; it is not an average.",
+			[]string{"session_name"},
+			nil,
+		),
+		PoolPagedBytes: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "pool_paged_bytes"),
+			"Number of bytes in the paged pool, an area of system memory (physical memory used by the operating system) for objects that can be written to disk when they are not being used. This property displays the last observed value only; it is not an average.",
+			[]string{"session_name"},
+			nil,
+		),
+		PrivateBytes: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "private_bytes"),
+			"Current number of bytes this process has allocated that cannot be shared with other processes.",
+			[]string{"session_name"},
+			nil,
+		),
+		ThreadCount: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "thread_count"),
+			"Number of threads currently active in this process. An instruction is the basic unit of execution in a processor, and a thread is the object that executes instructions. Every running process has at least one thread.",
+			[]string{"session_name"},
+			nil,
+		),
+		VirtualBytes: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "virtual_bytes"),
+			"Current size, in bytes, of the virtual address space the process is using. Use of virtual address space does not necessarily imply corresponding use of either disk or main memory pages. Virtual space is finite and, by using too much, the process can limit its ability to load libraries.",
+			[]string{"session_name"},
+			nil,
+		),
+		VirtualBytesPeak: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "virtual_bytes_peak"),
+			"Maximum number of bytes of virtual address space the process has used at any one time. Use of virtual address space does not necessarily imply corresponding use of either disk or main memory pages. Virtual space is finite and, by using too much, the process might limit its ability to load libraries.",
+			[]string{"session_name"},
+			nil,
+		),
+		WorkingSet: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "workingset"),
```

Comment:
Needs a unit, and we have separated it into `working_set` elsewhere
```suggestion
			prometheus.BuildFQName(Namespace, subsystem, "working_set_bytes"),
```